march_pdb_state_controller/src/march_pdb_state_controller.cpp

```diff
@@ -50,13 +46,9 @@ void MarchPdbStateController::turnLowVoltageNetOnOrOffCallBack(const std_msgs::I
 bool MarchPdbStateController::init(march_hardware_interface::MarchPdbStateInterface* hw, ros::NodeHandle& root_nh,
                                    ros::NodeHandle& controller_nh)
 {
-  ROS_INFO("init MarchPdbStateController");
   // Get all temperature_sensors from the hardware interface
   const std::vector<std::string>& pdb_state_names = hw->getNames();
   for (unsigned i = 0; i < pdb_state_names.size(); i++)
-    ROS_INFO("Got pdb state %s", pdb_state_names[i].c_str());
-
-  ROS_INFO("amount of pdb_state_names: %zu", pdb_state_names.size());
```

Comment:
Should these all be removed or rewritten to debugs?

Reply:
I don't think the logs are useful anymore, most of the prints were used to see how far the code would run